webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePattern.js

```diff
@@ -1,29 +1,65 @@
 import React from 'react';
-import { EmptyState as PfEmptyState } from 'patternfly-react';
+import { Icon } from 'patternfly-react';
+import {
+  Title,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+} from '@patternfly/react-core';
 import { emptyStatePatternPropTypes } from './EmptyStatePropTypes';
+import { translate as __ } from '../../../common/I18n';
+import './EmptyState.scss';
 
 const EmptyStatePattern = props => {
   const {
     documentation,
     action,
     secondaryActions,
-    icon,
     iconType,
+    icon,
     header,
     description,
   } = props;
 
+  const DocumentationBlock = () => {
+    if (
+      !documentation.hasOwnProperty('label') &&
+      !documentation.hasOwnProperty('buttonLabel')
+    ) {
+      const {
+        label = __('For more information please see'),
+        buttonLabel = __('documentation'),
+        url,
+      } = documentation;
+      return (
+        <span>
+          {label}
+          <a href={url}>{buttonLabel}</a>
+        </span>
+      );
+    }
+    return documentation;
+  };
```

Comment:
This does not work exactly as expected. If existed documentation is passed then it throws an error saying `documentation is undefined`

Reply:
maybe you meant to drop the `!` on lines 27,28 ?

Reply:
also IMO it can be `documentation.hasOwnProperty('label') || documentation.hasOwnProperty('buttonLabel')`

Reply:
> maybe you meant to drop the `!` on lines 27,28 ?

Error:
```
Uncaught Invariant Violation: Objects are not valid as a React child (found: object with keys {url}). If you meant to render a collection of children, use an array instead.
```

Reply:
got it: here we were returning the object `documentation`, instead we can do something like `return <React.Fragment />;`